lib/engine/step/g_18_mag/track.rb

```diff
@@ -104,6 +104,21 @@ def pay_tile_cost!(entity, tile, rotation, hex, spender, cost, extra_cost)
           end
           @log << "#{@game.sik.name} earns #{@game.format_currency(cost - extra_cost)}"
         end
+
+        # handle yellow OO tile
+        def update_token!(_action, _entity, tile, old_tile)
+          cities = tile.cities
+          if old_tile.paths.empty? &&
+              !tile.paths.empty? &&
+              cities.size > 1 &&
+              cities.flat_map(&:tokens).any?
```

Comment:
(token = cities.flat_map(&:tokens).find(&:itself))

then yo ucan delete the second one

Reply:
OK.